### DISCUSSION TITLE
default action for resume/pause/delete button

### DISCUSSION THREAD

**Discussion by flashlab** [VueTorrent/VueTorrent#1598] [Ideas]
For now, if no torrents selected, the RESUME/PAUSE button will apply to all torrents without prompt. It'll be good to have an option for users to choose the action.
![微信截图_20240324120016](https://github.com/VueTorrent/VueTorrent/assets/4102853/76b76c4b-e55c-4279-875f-36897bfacc5f)

**Comment by Larsluph** [comment]
It can be a great improvement indeed!

Make sure to create a new issue to see it implemented.

**Comment by Larsluph** [comment]
See #1607 for updates
